doc/man3/SSL_get_curve_name.pod

```diff
@@ -0,0 +1,41 @@
+=pod
+
+=head1 NAME
+
+SSL_get_curve_name - get name of the curve that was used for the key
+agreement of the current TLS session establishment
+
+=head1 SYNOPSIS
+
+ #include <openssl/ssl.h>
+
+ const char *SSL_get_curve_name(SSL *s);
+
+=head1 DESCRIPTION
+
+SSL_get_curve_name() returns the name of the curve that was used for
+the key agreement of the current TLS session establishment.
+
+
+=head1 RETURN VALUES
+
+SSL_get_curve_name() returns the name of the curve that was used for
+the key agreement of the current TLS session establishment.
+
+Note that the return value is valid only during the lifetime of the
+SSL object I<ssl>.
```

Comment:
What happens if the handshake hasn't been initiated or completed yet? If this returns NULL in that case, it would be good to mention it.

Reply:
Good point, I'm not actually sure if it would return NULL or just break. I will make sure it returns NULL and update the doc to reflect it.